### DISCUSSION TITLE
Emojis

### DISCUSSION THREAD

**Discussion by flaviozluca** [rspamd/rspamd#3834] [Q&A]
Hello, everyone!

Please, is it possible to add emojis to be detected as I add words to my blocklist? I tried with no success. These emojis are quite common, from what we can see, just in spams. So it would be a nice way to add up scoring.

Examples:

❗ ✉

We tried to add the envolve to be detected as Unicode, as such: \x{2709}

However, that didn't work. If it's not possible, would love to be able to add it as a suggestion.

Thanks!

**Comment by fatalbanana** [comment]
They are commonly used in ham as far as I see.

Unicode code points (and ranges) in regex work if you use the [/u modifier](https://rspamd.com/doc/modules/regexp.html#regular-expressions):

```
reconf['SUBJECT_CONTAINS_ENVELOPE'] = {
  re = [[Subject=/\x{2709}/Hu]],
  score = 0.0,
}

reconf['SUBJECT_CONTAINS_DINGBATS'] = {
  re = [[Subject=/[\x{2700}-\x{27bf}]/Hu]],
  score = 0.0,
}
```

**Reply by gtech99** [reply]
what about in maps ?
